### ISSUE TITLE
running chewbbaca says "list index out of range"

### ISSUE THREAD

**Issue by tseemann** [B-UMMI/chewBBACA#23]
```
% chewBBACA.py

chewBBACA version 2.0.16 by Mickael Silva at https://github.com/B-UMMI/chewBBACA
email contact: mickaelsilva@medicina.ulisboa.pt


list index out of range

        USAGE : chewBBACA.py [module] -h
```

**Comment by jacarrico** [comment]
Already on my to do list to do a fix. Thanks for the report

**Comment by mickaelsilva** [comment]
Corrected on 2.0.17.
